src/conversations/Conversations.ts

```diff
@@ -408,6 +430,37 @@ export default class Conversations {
     )
   }
 
+  async export(): Promise<ConversationExport[]> {
+    const conversations = await this.list()
+    return conversations.map((convo) => convo.export())
+  }
+
+  async import(convoExports: ConversationExport[]): Promise<number> {
```

Comment:
another dumb q: convos are exported in sorted order (`createdAt`) implicitly but there is no such requirement on import right? Just trying to think of any implicit invariants that could be gotchas in the future e.g. relied upon some ordering in exported data, ordering changes, imports break etc

Reply:
The reason for that being that we merge together `v1` and `v2` conversations and want the final results to have a consistent sort.

Reply:
Awesome, thanks!